cornedbeef/src/main/java/com/swiftkey/cornedbeef/PunchHoleCoachMark.java

```diff
@@ -235,6 +284,26 @@ public PunchHoleCoachMarkBuilder setOverlayColor(int overlayColor) {
         }
 
         /**
+         * Set the layout information for the content inside the coach mark, allowing explicit
+         * placement and sizing of the content with respect to the punch hole
+         *
+         * @param contentWidth the width of the content - defaults to {@link MATCH_PARENT}
+         * @param contentHeight the height of the content - defaults to {@link WRAP_CONTENT}
+         * @param contentPositioning where to place the content - defaults to
+         *  {@link POSITION_CONTENT_AUTOMATICALLY}
+         */
+        public PunchHoleCoachMarkBuilder setContentLayoutParams(
+                final int contentWidth,
+                final int contentHeight,
+                @PunchMarkContentPosition final int contentPositioning) {
+
+            this.contentWidth = contentWidth;
```

Comment:
Why do we need the content width and height? If we pass the view as a parameter, can't it have the width and height already set?